explorer/src/main/java/com/scalar/client/tool/explorer/command/Validate.java

```diff
@@ -53,8 +54,11 @@ public void run() {
         explorer -> {
           try {
             for (String assetId : assetIds) {
-              explorer.validate(assetId);
-              System.out.println(assetId + " is not tampered");
+              if(explorer.validate(assetId).getCode().equals(StatusCode.OK)) {
+                  System.out.println(assetId + " is not tampered");
+              } else {
+                  System.out.println(assetId + " not found");
```

Comment:
why asset not found if it is not OK ?
I think it should check if it returns `ASSET_NOT_FOUND` .
Also, it should check if it returns any validation errors.

Reply:
@feeblefakie san,
Thank you for review comment.
Sorry, It's my mistake.
Source code updated